src/main/java/com/datadoghq/sketch/ddsketch/mapping/CubicallyInterpolatedMapping.java

```diff
@@ -73,6 +73,8 @@
  */
 public class CubicallyInterpolatedMapping extends LogLikeIndexMapping {
 
+  private static final double CORRECTING_FACTOR = 7 / (10 * Math.log(2));
```

Comment:
you can use `C` here right?

Reply:
Yes, done in 80dbddd9df7e2e1a4aa50f7852e27808bcb86c5f.